lib/mp4box-player/src/Mp4boxPlayer.js

```diff
@@ -733,7 +746,7 @@ export class Mp4boxPlayer {
         sb.ms = ms;
         sb.id = track_id;
         this.mp4boxfile.setSegmentOptions(track_id, sb, {
-          nbSamples: this.config.segmentSize,
+          nbSamples: this.videoFps * this.SEGMENT_DURATION_SECS  //Consider changing to this.SEGMENT_DURATION_SECS/2  if prod needs more latency improvements later.
```

Comment:
During your testing were there any bad increased lags due to this new equation update?

Currently we are doing `FPS * 3secs`

Previous we were doing `FPS * 3/2 = FPS = 1.5secs`

Reply:
yes no change visibly. But /2 would have less memory footprint, a few "ms" improvement.